xml-conduit/Text/XML/Stream/Render.hs

```diff
@@ -72,6 +73,9 @@ data RenderSettings = RenderSettings
     , rsAttrOrder  :: Name -> Map.Map Name Text -> [(Name, Text)]
       -- ^ Specify how to turn the unordered attributes used by the "Text.XML"
       -- module into an ordered list.
+    , rsUseCDATA   :: Content -> Bool
+      -- ^ Determines if for a given text content the renderer should use a
+      -- CDATA node.
```

Comment:
Should include a comment that the default in `False` and a `@since` indicating the version this was added.